src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/InstructionDecoderTests.cs

```diff
@@ -282,6 +282,30 @@ static void M(params int[] x)
                 GetName(source, "C.M", DkmVariableInfoFlags.Types | DkmVariableInfoFlags.Names));
         }
 
+        [Fact, WorkItem(1154945, "DevDiv")]
+        public void GetNameIncorrectNumberOfArgumentValues()
+        {
+            var source = @"
+class C
+{
+    void M(int x, int y)
+    {
+    }
+}";
+
+            Assert.Equal(
+                "C.M(int x, int y)",
+                GetName(source, "C.M", DkmVariableInfoFlags.Types | DkmVariableInfoFlags.Names, argumentValues: new string[] { }));
```

Comment:
Ideally, I would have liked that (from the perspective of being more terse), but the helper takes multiple, optional array arguments, so I'd rather keep it this way (less confusing what you're calling).